podman/tinyproxy/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+
+# Vars
+ENV tpVersion 1.10.0
+ENV tpRelease tinyproxy-${tpVersion}
+
+# Default port
+EXPOSE 8888/tcp
+
+# Download, decompress, install and clean up
+RUN microdnf install gcc make gzip tar wget && \
+	wget https://github.com/tinyproxy/tinyproxy/releases/download/${tpVersion}/${tpRelease}.tar.gz && \
+	tar xzvf ${tpRelease}.tar.gz -C /tmp && \ 
+  cd /tmp/${tpRelease} && \
+	./configure && \
+	make && \
+	make install && \
+	cd .. && \
+	rm -rf ${tpRelease} ${tpRelease}.tar.gz && \
+	microdnf remove gcc make gzip tar wget && \
+	microdnf clean all
+
+# Configuration
+COPY ./conf/ /usr/local/etc/tinyproxy/
```

Comment:
Not a volume?

Reply:
Sort of!  This is default image config.  Volume overrides.